src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -118,6 +122,23 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
     }
   }
 
+  async validateTTLSeconds(ttl: number): Promise<void> {
+    try {
+      if (ttl < MIN_TTL_SECONDS) {
+        throw new Error(
+          `${ttl} is less than the minimum ttlSeconds requirement of ${MIN_TTL_SECONDS}`,
+        );
+      }
+      if (ttl > MAX_TTL_SECONDS) {
+        throw new Error(
+          `${ttl} is more than the maximum ttlSeconds requirement of ${MAX_TTL_SECONDS}`,
+        );
+      }
+    } catch (error: any) {
+      throw error;
+    }
+  }
```

Comment:
see comment above - this provider is meant for API calls to arweave/warp/cache and not util validations. move to separate function